dnf-behave-tests/features/microdnf/install7.feature

```diff
@@ -0,0 +1,14 @@
+@no_installroot
+Feature: Install package
+
+@bz1691353
+Scenario: Install an RPM without null lines
+  Given I use repository "dnf-ci-fedora"
+   When I execute microdnf with args "install lame"
```

Comment:
The test is run on the host - not in the install root. That said, all other repositories already enabled on host (like fedora, updates...) are used which makes the test considerably slower (need to sync all metadata for those repositories) and unable to run without the network (which is the way how gating tests are run).
You can solve this by removing all current repo files as first steps (see also other microdnf/install*.feature files in the master branch):
```
Background:
Given I delete file "/etc/dnf/dnf.conf" 
  And I delete file "/etc/yum.repos.d/*.repo" with globs
```

Reply:
Thank you for the explanation.